app/utils/AddATrusteeViewHelper.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import models.{AllTrustees, LeadTrustee, LeadTrusteeIndividual, LeadTrusteeOrganisation, TrusteeType}
+import play.api.i18n.Messages
+import viewmodels.addAnother.{AddRow, AddToRows}
+
+class AddATrusteeViewHelper(trustees: AllTrustees)(implicit messages: Messages) {
+
+  private def render(trustee : (TrusteeType, Int)) : AddRow = {
+
+    trustee match {
+      case (TrusteeType(Some(trusteeInd), None), index) =>
+        AddRow(
+          name = trusteeInd.name.displayName,
+          typeLabel = messages(s"entities.trustee.individual"),
+          changeUrl = "#",
+          removeUrl = controllers.trustee.routes.RemoveIndividualTrusteeController.onPageLoad(index).url
+        )
+      case (TrusteeType(None, Some(trusteeOrg)), index) =>
+        AddRow(
+          name = trusteeOrg.name,
+          typeLabel = messages(s"entities.trustee.organisation"),
+          changeUrl = "#",
+          removeUrl = controllers.trustee.routes.RemoveIndividualTrusteeController.onPageLoad(index).url
```

Comment:
Check this?

Reply:
@jassalrichy This will need to be it's own controller due to the change in the name structure for rendering